website/docs/how-to.md

```diff
@@ -270,9 +270,15 @@ targetFormats:
 datasets:
   -
     tableBasePath: file:///tmp/iceberg-dataset/people
-    dataBasePath: file:///tmp/iceberg-dataset/people/data
+    tableDataPath: file:///tmp/iceberg-dataset/people/data
     tableName: people
 ```
+:::note Note:
+Iceberg sourceFormat requires `tableDataPath` to be specified because Hudi expects the partition folders to be first
```

Comment:
What does `sourceFormat` mean here? It might be helpful to use common terms or provide references for beginners who are reading the docs. 
Also, as far as I remember, dataPath was introduced for targets, not for source clients. In other words, it is not used by Iceberg source, rather by Hudi and Delta targets.

Reply:
@ashvina it was introduced as a parameter describing the source that is passed on to the targets so they can create their metadata in the correct location. 

I do think this note is diving a bit too deep into the weeds for a how-to.

Reply:
Also @ashvina sourceFormat is the name of the field in the yaml so it is consistent within this context.

Reply:
@the-other-tim-brown do you think we should remove the note?

Reply:
Yes or just simplify it by saying that we currently require it if the base path of the table is not the same as the path to the data

Reply:
> Also @ashvina sourceFormat is the name of the field in the yaml so it is consistent within this context.
My bad, I was assuming the note talks about the SourceClient.